components/Settings/ProfileTab.js

```diff
@@ -7,6 +7,10 @@ const Wrapper = s.div`
   font-size: 18px;
 `
 
+const Empty = s.span`
+  color: #888;
```

Comment:
could this be a color from the `colors.js` file? feel free to add a color if we could use it but this seems like it'd be close to `DARK_GRAY`